portfolio/src/main/webapp/script.js

```diff
@@ -73,6 +73,23 @@ function loadComments() {
   });
 }
 
+/**
+ * Add a new comment to the page from form input
+ */
+function addComment() {
+  const commentContainer = document.getElementById('comment-container');
+  const nameInput = document.getElementById("name-input").value;
+  const commentInput = document.getElementById("comment-input").value;
+  const request = new Request(
+    `/data?name-input=${nameInput}&comment-input=${commentInput}`,
+    {method: 'POST'}
+  );
+  fetch(request).then(() => {
+    const newComment = {name:nameInput, text:commentInput};
+    commentContainer.appendChild(newCommentBox(newComment));
```

Comment:
It is better practice to just call `loadComments()` here. If the back-end has a bug and doesn't actually store it in the database, then here it looks like it successfully worked. Just in generally, best to show canonical data.